src/Kernel-Tests/CompiledCodeTest.class.st

```diff
@@ -30,6 +30,18 @@ CompiledCodeTest >> testAccessesRef [
 	self assert: (writeMethod accessesRef: classVar).
 ]
 
+{ #category : #'tests - accessing' }
+CompiledCodeTest >> testBytecodeSizeForPc [
+
+	| method |
+	method := self class >> #method1.
+
+	self
+		assert: (method bytecodeSizeForPc: method initialPC)
+		equals:
+		(method encoderClass bytecodeSize: (method at: method initialPC))
```

Comment:
Again, this test really depends on the input, and different inputs do different things. Maybe they need different tests...

Reply:
Probably we were started to see.